### ISSUE TITLE
[QUESTION] get arguments value

### ISSUE THREAD

**Issue by aemilc** [tiangolo/typer#104]
### First check

* [X] I used the GitHub search to find a similar issue and didn't find it.
* [X] I searched the Typer documentation, with the integrated search.
* [X] I already searched in Google "How to X in Typer" and didn't find any information.
* [x] I already searched in Google "How to X in Click" and didn't find any information.

### Description

Does typer collect all the values given to a dictionary or a tuple?

Thanks

**Comment by aschreyer** [comment]
`context.params` has all the _raw_ values from the CLI.

**Comment by aemilc** [comment]
> `context.params` has all the _raw_ values from the CLI.

Thank you! 
Is this coming from click? 
The documentation is really well done, but I cannot find how to do some stuff.
I came from argparse, but that's my first time with other CLI. 
For example, how can I have a feature like "choice" to allow only certain values for an option? 
I did it with a callback now, but I wonder if this is the right way. 
Where I can find all the features of typer?

Thanks
